pkg/dataplane/server/dataplane.go

```diff
@@ -0,0 +1,77 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	cluster "github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3"
+	listener "github.com/envoyproxy/go-control-plane/envoy/config/listener/v3"
+	"github.com/go-chi/chi"
+	"github.com/sirupsen/logrus"
+
+	"github.com/clusterlink-net/clusterlink/pkg/util"
+)
+
+// Dataplane implements the server and api client which sends authorization to the control plane
+type Dataplane struct {
+	ID                 string
+	peerName           string
+	router             *chi.Mux
+	apiClient          *http.Client
+	parsedCertData     *util.ParsedCertData
+	controlplaneTarget string
+	clusterMap         map[string]*cluster.Cluster
```

Comment:
there is no need, IMO, to add the type name to variable names (`clusterMap`, `listenerChan`, etc).

Reply:
Agree 👍🏽